### PR TITLE
Doc fixes for query from deep storage and MSQ

### PR DESCRIPTION
**Pull request by cryptoe**
Minor updates to the documentation. 
* Added pre requisites. 
* Removed a known issue in MSQ since its no longer valid.

### CODE REVIEW

**Review by gianm — APPROVED** [review]

File: docs/querying/query-from-deep-storage.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ title: "Query from deep storage"
 
 Druid can query segments that are only stored in deep storage. Running a query from deep storage is slower than running queries from segments that are loaded on Historical processes, but it's a great tool for data that you either access infrequently or where the low latency results that typical Druid queries provide is not necessary. Queries from deep storage can increase the surface area of data available to query without requiring you to scale your Historical processes to accommodate more segments.
 
+## Pre-requisites
+* Please enable the multi-stage query extension before you begin. See [enable multi-stage query](../multi-stage-query/index.md#load-the-extension) for more information.
```
Comment (317brian) [inline docs/querying/query-from-deep-storage.md:28]:
```suggestion

Query from deep storage requires the Multi-stage query (MSQ) task engine. Load the extension for it if you don't already have it enabled before you begin. See [enable MSQ](../multi-stage-query/index.md#load-the-extension) for more information.
```

**Review by 317brian — APPROVED** [review]
Minor nit but LGTM otherwise

File: docs/querying/query-from-deep-storage.md [hunk 2]
```diff
@@ -24,6 +24,9 @@ title: "Query from deep storage"
 
 Druid can query segments that are only stored in deep storage. Running a query from deep storage is slower than running queries from segments that are loaded on Historical processes, but it's a great tool for data that you either access infrequently or where the low latency results that typical Druid queries provide is not necessary. Queries from deep storage can increase the surface area of data available to query without requiring you to scale your Historical processes to accommodate more segments.
 
+## Pre-requisites
```
Comment (317brian) [inline docs/querying/query-from-deep-storage.md:27]:
```suggestion
## Prerequisites
```

File: docs/tutorials/tutorial-query-deep-storage.md [hunk 3]
```diff
@@ -31,6 +31,8 @@ To run the queries in this tutorial, replace `ROUTER:PORT` with the location of
 
 For more general information, see [Query from deep storage](../querying/query-from-deep-storage.md).
 
+If you are trying this feature on an existing cluster, make sure query from deep storage [pre-requisites](../querying/query-from-deep-storage.md#pre-requisites) are met.
```
Comment (317brian) [inline docs/tutorials/tutorial-query-deep-storage.md:34]:
```suggestion
If you are trying this feature on an existing cluster, make sure query from deep storage [pre-requisites](../querying/query-from-deep-storage.md#prerequisites) are met.
```

**Pull request merged**